spec/built-in-tiles-spec.coffee

```diff
@@ -67,10 +67,19 @@ describe "Built-in Status Bar Tiles", ->
       it "calls relativize with the remote URL on shift-click", ->
         spy = spyOn(atom.project, 'relativize').andReturn 'remote_file.txt'
         event = new MouseEvent('click', shiftKey: true)
-        fileInfo.currentPath.dispatchEvent(event)
+        fileInfo.dispatchEvent(event)
         expect(atom.clipboard.read()).toBe 'remote_file.txt'
         expect(spy).toHaveBeenCalledWith 'remote://server:123/folder/remote_file.txt'
 
+    describe "when file info tile is clicked", ->
+      it "copies the absolute path into the clipboard if available", ->
+        waitsForPromise ->
+          atom.workspace.open('sample.txt')
+
+        runs ->
+          fileInfo.click()
+          expect(atom.clipboard.read()).toBe fileInfo.getActiveItem().getPath()
+
     describe "when buffer's path is clicked", ->
```

Comment:
I don't think we need this test anymore, as the info tile element includes the buffer path element.